package.json

```diff
@@ -151,9 +162,10 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run prepush",
-      "pre-commit": "npm run security-checks"
+      "commit-msg": "./node_modules/.bin/commitlint -E HUSKY_GIT_PARAMS",
+      "pre-commit": "npm run lint && npm run security-checks",
+      "pre-push": "npm run prepush"
     }
   },
   "description": "CVS VTM Application"
-}
+}
```

Comment:
@JonLazariniDD i think the settings in your editor is removing the extra line in files